lib/state/ui/reducer.ts

```diff
@@ -52,17 +50,20 @@ const simperiumConnected: A.Reducer<boolean> = (state = false, action) =>
     ? action.simperiumConnected
     : state;
 
-const visiblePanes: A.Reducer<string[]> = (
-  state = defaultVisiblePanes,
-  action
-) => {
-  if ('TAG_DRAWER_TOGGLE' === action.type) {
-    return action.show
-      ? union(state, ['tagDrawer'])
-      : difference(state, ['tagDrawer']);
+const trash: A.Reducer<boolean> = (state = false, action) => {
```

Comment:
can we call this something other than `trash`? maybe even keep `showTrash` or `viewingTrash` or `trashOpen`?

I find `trash` (especially in `mapStateToProps`) to be a wee bit confusing

Reply:
done